src/lightning/pytorch/loggers/logger.py

```diff
@@ -111,10 +109,8 @@ def merge_dicts(  # pragma: no cover
         default_func:
             Default function to aggregate keys, which are not presented in the
             `agg_key_funcs` map.
-
     Returns:
         Dictionary with merged values.
-
```

Comment:
The changes to this file should be reverted. They don't seem to relate to the PR's purpose